pkg/manager/manager_test.go

```diff
@@ -1303,6 +1303,50 @@ var _ = Describe("manger.Manager", func() {
 				Expect(ok).To(BeTrue())
 			})
 
+			It("should serve metrics in an extra registry", func() {
+				extraRegistry := prometheus.NewRegistry()
```

Comment:
The prometheus registry itself implements `Collector` so could instead just do a `metrics.Registry.MustRegister(extraRegistry)` or not?